akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterSharding.scala

```diff
@@ -446,6 +452,16 @@ class ClusterSharding(system: ExtendedActorSystem) extends Extension {
   }
 
   /**
+    * Scala API: get all defined sharding typeName's.
+    */
+  def shardTypeNames: immutable.Set[String] = regions.keySet().asScala.toSet
+
+  /**
+    * Java API: get all defined sharding typeName's.
```

Comment:
Unfortunately our formatter prefers these lines to be formatted differently. Compiling from sbt should automatically fix the formatting, could you try that?